### DISCUSSION TITLE
Can't fetch results when VPN is connected

### DISCUSSION THREAD

**Discussion by RelicCornhusk** [searx/searx#2826] [Installation issue: Docker] [answered]
This might be a dumb question, but could anyone help me figure why Searx does not return any results when my VPN is connected? 
For context, I'm running a private instance through a Docker container. Currently using the Linux distribution Pop!_OS 20.10 and connected to NordVPN. I've tried tweaking all of the VPN's settings to see if it would make a difference (protocol, killswitch, firewall, etc.) but still no results. I consistently get " Error! Engines cannot retrieve results." saying all search engines timed out, and the moment I switch off the VPN it starts working again. I suppose it has to do with my IP address changing, but how can I configure Searx or Docker to continue working when I'm using a VPN? Much appreciated.

**Comment by RelicCornhusk** [comment]
So simply restarting the computer did it for me. Really sorry for taking your time. I would close/delete this issue but apparently GitHub won't let me, so feel free to do so.

**Reply by unixfox** [reply]
It's not a GitHub issue but a discussion and it's fine to keep it opened. Or if you really want to have it deleted then I can do it anyway.

**Reply by RelicCornhusk** [reply]
No problem leaving it open then! Maybe someone can offer some insight as to what was happening. Thanks!

**Comment by RelicCornhusk** [comment, marked as answer]
Ok, I figured out how to solve this problem in case anyone also goes through this. At first restarting worked for some reason, but now the problem is back, so what I did is I edited the section in settings.yml that deals with Source IPs. You need to uncomment that section and add "0.0.0.0" or "::" (depending if you use ipv4 or ipv6)  and it will accept requests from any local IPV4 addresses. This is described [here](https://searx.github.io/searx/admin/settings.html).
